systemtest/src/test/java/io/strimzi/systemtest/watcher/AbstractNamespaceST.java

```diff
@@ -98,4 +331,41 @@ void deployKafkaConnectorWithSink(ExtensionContext extensionContext, String clus
 
         KafkaConnectUtils.waitForMessagesInKafkaConnectFileSink(testStorage.getNamespaceName(), kafkaConnectPodName, Constants.DEFAULT_SINK_FILE_PATH, "99");
     }
+
+    /**
+     * Helper method which deploys Kafka Cluster and Scraper Pod in primary namespace. It is supposed to be called once there is a running Cluster Operator.
+     *
+     * @param extensionContext extension context to create kafka cluster accordingly.
+     */
+    protected void deployAdditionalGenericResourcesForAbstractNamespaceST(ExtensionContext extensionContext) {
```

Comment:
and I assume the method should also be `final`. Or is there an intention to override it in subclasses?

Reply:
true also to be final 👍